scripts/check-sse4_2.sh

```diff
@@ -44,7 +44,9 @@ for dir in $dirs; do
             # f2.*0f 38 is the opcode of `crc32`, see Intel® SSE4 Programming Reference
             found=0
             for sym in $fast_crc32; do
-                if [[ `objdump --disassemble="$sym" $dirfile 2> /dev/null | grep ".*f2.*0f 38.*crc32"` ]]; then
+		read -r start stop <<<$(nm "$dirfile" | grep -A1 "$sym" | awk '{printf("0x"$1" ")}')
+		stop=${stop:-0xFFFFFFFFFFFFFFFF} # This is the last symbol
```

Comment:
I believe the case where Fast_CRC32 is the last symbol is impossible. At least on Linux I see a special `_end` symbol as the last symbol, and this script only runs on Linux.

So I don't think this defensiveness is necessary for the reason stated in the comment. That said, there are so many ways that bash scripts can screw up that I think it is good policy generally for bash scripts to be defensive.

I might replace this line with

```
test -n "$start" || echo "start undefined" && exit 1
test -n "$stop" || echo "stop undefined" && exit 1
```

Reply:
Given that the `start` got from `nm`, it must exist. if the next symbol should not be the last, I prefer to omit the two `test` statement.